C/cif_api/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cif_api"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/COMCIFS/cif_api/archive/refs/tags/v0.4.2.tar.gz", "803fa1d0525bb51407754fa63b9439ba350178f45372103e84773ed4871b3924")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/cif_api-*
+
+update_configure_scripts
+
+if [[ "${target}" == *-linux-* ]]; then
+# Hint to find libstdc++, required to link against C++ libs when using C compiler
+    if [[ "${nbits}" == 32 ]]; then
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib"
+    else
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib64"
+    fi
+fi
```

Comment:
I don't think this is necessary after https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/164

Reply:
This made me realize I had not updated my Project or Manifest file in a while